Doc/faq/programming.rst

```diff
@@ -1840,7 +1848,7 @@ identity tests.  This prevents the code from being confused by objects such as
 ``float('NaN')`` that are not equal to themselves.
 
 For example, here is the implementation of
-:meth:`collections.abc.Sequence.__contains__`::
+:meth:`!collections.abc.Sequence.__contains__`::
```

Comment:
```suggestion
:meth:`collections.abc.Sequence.__contains__`::
```

Arguably, perhaps best to keep this as a known-broken link, as ideally it _should_ resolve and it will, if/when it is added.